torch/jit/__init__.py

```diff
@@ -1804,10 +1808,6 @@ def _register_builtin(fn, op):
 def _find_builtin(fn):
     return _get_builtin_table().get(id(fn))
 
-
-_register_builtin(len, 'aten::len')
```

Comment:
this one was deleted since it's defined in `compiler.cpp`

Reply:
I think these are intended that way so that you can assign a variable a builtin function and it will still compile. `l_fn = len ... l_fn(tensor)`

However, emitting the builtins as an aten op instead of their full name kind of messes things up because it won't work on a Class Type.

Reply:
Yeah, maybe we could move our global => SugaredValue mapping to Python since everything gets de-sugared internally anyways, so we'd never need to resolve the globals without Python present